components/atoms/Icon/Icon.js

```diff
@@ -19,13 +19,25 @@ function sizeToPx(size) {
   return sizeToRem[size] * parseFloat(tailwindConfig.theme.fontSize['root-em'])
 }
 
+/**
+ * Render the Icon component.
+ *
+ * @param {object}  props            props.
```

Comment:
```suggestion
 * @param {object}  props            The component attributes as props.
```

Reply:
🤦 Great catch, I'll get this updated. I must have overlooked that.